src/pages/in/like/_id/index.vue

```diff
@@ -251,7 +252,18 @@ export default {
         this.like();
       }
 
-      if (this.isMaxLike) {
+      if (
+        this.isMaxLike
+        && (
+          !this.isSubscribed
+          || this.isTrialSubscriber
+          || (
+            this.isSubscribed
+            && !this.isTrialSubscriber
+            && (!checkIsMobileClient() || checkIsTrustClient())
```

Comment:
add `const isPaidUser = (!this.isSubscribed || this.isTrialSubscriber);` to look less messy?